.github/workflows/clippy.yaml

```diff
@@ -0,0 +1,22 @@
+name: Clippy
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+
+jobs:
+  lints:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Rust toolchain and cache
+        uses: actions-rust-lang/setup-rust-toolchain@v1.5.0
+        with:
+          rustflags: ""
+
+      - name: Clippy
+        run: cargo clippy -- --all-features -Dwarnings
```

Comment:
This should probably be checking all targets in the workspace.

Reply:
It should already be doing that.

Reply:
I confirmed it is already checking all the members (except the one that is currently excluded and known-broken, that's a matter for a different issue).